libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ActionScope.cs

```diff
@@ -74,7 +74,7 @@ public override string GetVersion()
                 }
             }
 
-            return Convert.ToBase64String(SHA256.Create().ComputeHash(Encoding.UTF8.GetBytes(sb.ToString())));
+            return StringUtils.Hash(sb.ToString());
```

Comment:
Naming: Base64DecodedHash would be more accurate but definitely wordy. Seems like Hash is an over simplification but at the same time Base64DecodedHash sounds pretty long. Up to you, happy to brainstorm.